apis/v0/plugin.go

```diff
@@ -69,6 +70,10 @@ func (v *ExplanationValidator) Validate(_ context.Context, req *ValidateJustific
 	return &ValidateJustificationResponse{Valid: true}, nil
 }
 
+func (v *ExplanationValidator) GetUIData(_ context.Context, req *GetUIDataRequest) (*UIData, error) {
+	return &UIData{}, nil
```

Comment:
Should return the following
Display name: "Explanation"
Hint: "A justification reason in free-form text."

Reply:
Updated with default value. Thanks!